sources/AppBundle/Controller/Admin/Event/StatsAction.php

```diff
@@ -55,16 +64,34 @@ public function __invoke(Request $request)
         $id = $request->query->get('id');
         $comparedEventId = $request->query->get('compared_event_id');
 
+        if(!$id && !$comparedEventId) {
+            $compareData =  $request->query->get('compare_event');
+            $id = $compareData['id'];
+            $comparedEventId = $compareData['compared_event_id'];
+        }
+
+        dump($id, $comparedEventId);
+
         $event = $this->eventActionHelper->getEventById($id);
 
         $legacyInscriptions = $this->legacyModelFactory->createObject(Inscriptions::class);
 
-        $comparedSerieName = 'n-1';
         if ($comparedEventId) {
             $comparedEvent = $this->eventActionHelper->getEventById($comparedEventId, false);
             $comparedSerieName = $comparedEvent->getTitle();
+        } elseif($id !== null) {
+            $comparedEventId = $this->forum->obtenirForumPrecedent($id);
+            $comparedEvent = $this->eventActionHelper->getEventById($comparedEventId, false);
+            $comparedSerieName = $comparedEvent->getTitle();
+        } else {
+            $comparedSerieName = 'n-1';
```

Comment:
ici on pourrait se passer de ce elseif, et ne jamais afficher le "n-1", ça permettrais d'être plus lisible en étant toujours explicite et affichant toujours le nom de l'événement comparé (ce qui est maintenant possible vu qu'on a remonté l'appel à $this->forum->obtenirForumPrecedent($id)